.github/workflows/release.yml

```diff
@@ -0,0 +1,189 @@
+name: "Release"
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Any tag?? `tag` starting with `v` would better.

Reply:
We don't have tags started with "v".

Reply:
Okay